src/init.cpp

```diff
@@ -22,6 +22,7 @@
 #include <httpserver.h>
 #include <index/blockfilterindex.h>
 #include <index/txindex.h>
+#include <init/common.h>
 #include <interfaces/chain.h>
 #include <interfaces/node.h>
 #include <key.h>
```

Comment:
000e94b26c7340dfc834299d75848c34c73b3128: Could remove pubkey/key include, now that they are no longer used?

Reply:
pubkey not addressed?

Reply:
re: https://github.com/bitcoin/bitcoin/pull/21732#discussion_r618106018

> pubkey not addressed?

Yes I dropped `<key.h>` but didn't see any `<pubkey.h>` so assumed it was mistake. Are we looking at the same file https://github.com/ryanofsky/bitcoin/blob/pr/initc.3/src/init.cpp?

Reply:
Ah, so someone forgot to include it before the change. All good.